Source/tusdotnet/Models/DefaultTusConfiguration.cs

```diff
@@ -59,6 +59,11 @@ public class DefaultTusConfiguration
         /// </summary>
         public virtual long? MaxAllowedUploadSizeInBytesLong { get; set; }
 
+        /// <summary>
+        /// Enable the usage of System.IO.Pipelines library. Implement ITusPipelineStore to use it.
+        /// </summary>
+        public virtual bool EnablePipelines { get; set; }
```

Comment:
See comment on ITusPipelineStore. I don't think this is needed.

Reply:
Without that option, how should tus decide which version of AppendDataAsync should be called? Always use the pipelines version when available?

Reply:
Yes, that's how the design of interfaces in tusdotnet works. Basically `if(store is ITusPipelineStore pipelineStore) { /* do stuff */ }`